appveyor.yml

```diff
@@ -8,26 +8,26 @@ environment:
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       PLATFORM: x64
       BUILD_TYPE: Release
-      PYTHON_VERSION: 38
+      PYTHON_VERSION: 39
 
     - COMPILER: msvc
       GENERATOR: Visual Studio 15 2017
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
       PLATFORM: Win32
       BUILD_TYPE: Release
-      PYTHON_VERSION: 38
+      PYTHON_VERSION: 39
 
     - COMPILER: msvc
       GENERATOR: Visual Studio 14 2015 Win64
       PLATFORM: x64
       BUILD_TYPE: Release
-      PYTHON_VERSION: 38
+      PYTHON_VERSION: 39
 
     - COMPILER: msvc
       GENERATOR: Visual Studio 14 2015
       PLATFORM: Win32
       BUILD_TYPE: Release
-      PYTHON_VERSION: 38
+      PYTHON_VERSION: 39
 
     - COMPILER: mingw
       GENERATOR: MinGW Makefiles
```

Comment:
Think these MinGW builds also need Python version set to 3.9. Appears they use Python 2.7 otherwise